### PR TITLE
Settings content naming update

### PR DESCRIPTION
**Pull request by melissaperreault**
**Why are these changes introduced?**

Fixes https://github.com/Shopify/dawn/issues/985
Fixes https://github.com/Shopify/dawn/issues/986

**What approach did you take?**

- Update the final naming where possible
- Removed the usage of `Width`, it wasn't used anywhere
- Removed unused translations
- It is missing `Overlays`, this will be tackled with https://github.com/Shopify/dawn/issues/961
- It is missing `Interactive media controls`, it will be tackled once this is tackled. https://github.com/Shopify/dawn/issues/1051
- `Layout` will be updated from this comment https://github.com/Shopify/dawn/issues/925#issuecomment-995414537

**Demo links**

- [Editor](https://os2-demo.myshopify.com/admin/themes/127024234518/editor)

### CODE REVIEW

**Review by katycobb — COMMENTED** [review]
Hi Meli, couple of things:

- Under Cards, I don't think the "general" heading is needed. We don't use this elsewhere. 
- Dropdowns and pop-ups is missing its help text
- Under Icons, do we need the "decorative elements" header? I don't think it's necessary but you might have more context re: why it should be there! 


Otherwise the names look accurate.  If the other stuff isn't relevant to this ticket let me know and I can create a different one!

File: locales/en.default.schema.json
Comment (martinamarien) [inline locales/en.default.schema.json:325]: This line needs to be re-added

File: locales/en.default.schema.json
Comment (martinamarien) [inline locales/en.default.schema.json:2438]: This line needs to be removed.

**Comment by melissaperreault** [comment]
> Hi Meli, couple of things:

@katycobb Hey! Yes, this is not in this PR that I accounted for removing them because I was touching just the language file, we should see most of them revised under this PR https://github.com/Shopify/dawn/pull/1042 except for: "_Dropdowns and pop-ups is missing its help text_" --->>> This will be done in a follow up PR once one of them is merged. 🙏

**Review by ludoboludo — APPROVED** [review]
Looks good. Should we request translations ?

**Comment by melissaperreault** [comment]
> Looks good. Should we request translations ?

@ludoboludo  Not yet, I have additional changes I would like to apply once I merge https://github.com/Shopify/dawn/pull/1042

**Review by katycobb — COMMENTED** [review]
One nitpicky catch: Under `Media`, the heading says `Borders` but should be `Border` (no 's'). Otherwise all looks good!

**Comment by melissaperreault** [comment]
> One nitpicky catch: Under Media, the heading says Borders but should be Border (no 's'). Otherwise all looks good!

@katycobb Good catch thanks! I updated it! You can re-approve once you feel all is good!

File: locales/en.default.schema.json
Comment (ludoboludo) [inline locales/en.default.schema.json:203]: Almost feels likes if the only thing to edit about `Icons` is the color, it could potentially live in the `Colors` dropdown 🤔 
Just a thought.

**Review by ludoboludo — APPROVED** [review]

File: locales/en.default.schema.json
Comment (melissaperreault) [reply locales/en.default.schema.json:203]: Good observation, but there will be additional settings in the future under Icons such as the thickness of icons stroke  for instance.

**Pull request merged**
